client/index.html

```diff
@@ -12,8 +12,51 @@
     <link rel="shortcut icon" href="app/icon/carrot_logo.ico">
     <link rel="icon" type="image/png" href="app/icon/carrot_logo.png" sizes="32x32">
     <base href="/">
+    <style type="text/css">
```

Comment:
I'll put the loading dots in a class called `angular-pre-loading`. Just to make it clear what the code is for.
Even better: put it into a separate file, but that requires some webpack magic...